Dockerfile

```diff
@@ -17,4 +17,6 @@ WORKDIR /tmp/connect
 RUN bundle -j8
 ADD . /tmp/connect
 RUN chown -R nobody /tmp/connect
-RUN chmod +x /tmp/connect/docker/runall.sh
+RUN chmod +x /tmp/connect/docker/prophet_tests.sh /tmp/connect/docker/integration.sh
+ADD .oscrc /root/.oscrc
+ADD .regcode /root/.regcode
```

Comment:
I would go for getting it added in `RUN`, because we can save time on every run, and those files don't change often.